dev-util/bakefile/bakefile-0.2.11.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="Native makefiles generator"
+DESCRIPTION="Bakefile is cross-platform, cross-compiler native makefiles \
+generator. It takes compiler-independent description of build tasks as input \
+and generates native makefile (autoconf's Makefile.in, Visual C++ project, \
+bcc makefile etc.)."
+HOMEPAGE="https://bakefile.org"
+COPYRIGHT="2003-2007 Vaclav Slavik"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/vslavik/bakefile/releases/download/v$portVersion/bakefile-$portVersion.tar.gz"
+CHECKSUM_SHA256="be8b3ded51b4c86c3ece9898e21b5ab4ed474b7d82307b289c2e02f741b5b370"
+PATCHES="bakefile-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```

Comment:
Good news: bakefile builds fine on x86_gcc2!

If you agree, I can take care of updating your PR to mark `x86_gcc2` as tested, drop the support for secondary architectures, and drop the previous recipe.

Reply:
Sure, no problem! :+1: